content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,325 @@
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chown dd-agent:dd-agent \
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chmod 640 /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo systemctl restart datadog-fips-proxy
+   ```
+
+{{% /tab %}}
+
+{{% tab "RHEL and Fedora" %}}
+1. Run the following commands to install the FIPS proxy:
+   ```shell
+   yum makecache
+   yum install datadog-fips-proxy
+   ```
+
+1. The first time you perform an upgrade, copy the example configuration file to the appropriate location and restart the proxy. You do not need to copy the configuration in subsequent upgrades:
+   ```shell
+   sudo cp /etc/datadog-fips-proxy/datadog-fips-proxy.cfg.example \
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chown dd-agent:dd-agent \
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chmod 640 /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo systemctl restart datadog-fips-proxy
+   ```
+
+{{% /tab %}}
+
+{{% tab "SLES" %}}
+1. Run the following commands to install the FIPS proxy:
+   ```shell
+   zypper refresh datadog
+   zypper install datadog-fips-proxy
+   ```
+
+1. The first time you perform an upgrade, copy the example configuration file to the appropriate location and restart the proxy. You do not need to copy the configuration in subsequent upgrades:
+   ```shell
+   sudo cp /etc/datadog-fips-proxy/datadog-fips-proxy.cfg.example \
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chown dd-agent:dd-agent \
+      /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo chmod 640 /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+   sudo systemctl restart datadog-fips-proxy
+   ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Configure the Agent to use the FIPS proxy
+
+The Datadog FIPS proxy package comes pre-configured for use with the US1-FED datacenter. If you're upgrading an existing Datadog Agent, you must configure the Agent to use the proxy.
+
+To configure the Agent to use the FIPS proxy, set `fips.enabled` to `true` and `fips.https` to `false` in the [Agent configuration file][4]:
+
+```yaml
+fips:
+  enabled: true
+  https: false
+```
+
+The `fips` setting is available in Agent versions >= 7.41. When the setting is enabled, the Datadog Agent redirects all of its communications to the Datadog FIPS proxy. This setting ignores custom URL options, such as `dd_url`.
+
+The `https` option is set to false because the Agent uses HTTP to communicate with the proxy. The Datadog FIPS proxy is expected to run on the same host as the Datadog Agent and relies on the host's security for protection of that communication. Host security and hardening are customer responsibilities.
+
+<div class="alert alert-warning">The <code>fips.enabled</code> setting defaults to <code>false</code> in the Agent. It must be set to <code>true</code> to ensure all communications are forwarded through the Datadog FIPS proxy.</div>
```

Comment:
It should be noted here that Agent is not FIPS compliant without this flag set to true